presto-native-execution/testcontainers/README.md

```diff
@@ -0,0 +1,50 @@
+# Prestissimo: Functional Testing Using Containers
+
+This Java test framework allows you to run test cases by deploying Presto coordinator and worker nodes in containers.
```

Comment:
Nit : Add a link to any references for TestContainers which gives engineers more information about that framework.

Reply:
Added reference to [Testcontainers for Java](https://java.testcontainers.org/)